nexus/types/src/inventory/display.rs

```diff
@@ -429,7 +496,9 @@ fn display_sleds(
         writeln!(
             f,
             "    found at:    {} from {}",
-            sled.time_collected, sled.source
+            sled.time_collected
+                .to_rfc3339_opts(SecondsFormat::Millis, /* use_z */ true),
```

Comment:
Hah, I had made this exact same change before realizing it was enough and filing #8611 :+1: